apps/console/src/components/pages/protected/tasks/create-task/sidebar/task-details-sheet.tsx

```diff
@@ -347,14 +380,14 @@ const TaskDetailsSheet = () => {
                             />
                           </div>
                           <FormControl>
-                            <Textarea rows={7} id="details" {...field} className="w-full" />
+                            <PlateEditor onChange={handleDetailsChange} initialValue={taskData?.details ?? undefined} />
```

Comment:
```suggestion
                            <PlateEditor onChange={handleDetailsChange} initialValue={taskData?.details} />
```
??  is undefined check, no need to check for undefined and then set to undefined. if taskData or taskData.details is undefined, it will return undefined anyways